packages/auth/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@aerogearservices/auth",
+  "version": "1.0.0",
+  "description": "JavaScript Auth module for AeroGear services",
+  "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "author": "Aerogear <aerogear@googlegroups.com>",
+  "keywords": [
+    "aerogear",
+    "mobile"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "clean": "del coverage src/**/*.js src/**/*.map test/**/*.js test/**/*.map",
+    "build": "tsc",
+    "watch": "tsc --watch",
+    "test": "nyc mocha"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "nyc": {
+    "include": [
+      "src/**/*.ts"
+    ],
+    "extension": [
+      ".ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ],
+    "reporter": [
+      "lcov",
+      "text"
+    ],
+    "lines": 75,
+    "functions": 100,
+    "branches": 80
+  },
+  "devDependencies": {
+    "@types/chai": "4.1.2",
+    "@types/chai-as-promised": "7.1.0",
+    "@types/cordova-plugin-camera": "2.4.0",
+    "@types/lodash.find": "^4.6.3",
+    "@types/mocha": "^5.0.0",
+    "@types/node": "^9.4.6",
+    "@types/proxyquire": "1.3.28",
+    "@types/sinon": "^4.3.1",
+    "@types/uuid": "^3.4.3",
```

Comment:
You do not use this package along with lodash.find types above. Let's remove that.